### ISSUE TITLE
Crash in map fragment

### ISSUE THREAD

**Issue by BishopGIS** [nextgis/android_gisapp#492]
```
java.lang.NullPointerException
	at com.nextgis.mobile.fragment.MapFragment$4.onDismissed(MapFragment.java:630)
	at android.support.design.widget.Snackbar.onViewHidden(Snackbar.java:627)
	at android.support.design.widget.Snackbar.access$300(Snackbar.java:69)
	at android.support.design.widget.Snackbar$9.onAnimationEnd(Snackbar.java:582)
	at android.support.v4.view.ViewPropertyAnimatorCompatJB$1.onAnimationEnd(ViewPropertyAnimatorCompatJB.java:47)
	at android.view.ViewPropertyAnimator$AnimatorEventListener.onAnimationEnd(ViewPropertyAnimator.java:1030)
	at android.animation.ValueAnimator.endAnimation(ValueAnimator.java:1056)
	at android.animation.ValueAnimator.access$400(ValueAnimator.java:50)
	at android.animation.ValueAnimator$AnimationHandler.doAnimationFrame(ValueAnimator.java:644)
	at android.animation.ValueAnimator$AnimationHandler.run(ValueAnimator.java:660)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:761)
	at android.view.Choreographer.doCallbacks(Choreographer.java:574)
	at android.view.Choreographer.doFrame(Choreographer.java:543)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:747)
	at android.os.Handler.handleCallback(Handler.java:733)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:136)
	at android.app.ActivityThread.main(ActivityThread.java:5021)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:827)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:643)
	at dalvik.system.NativeStart.main(Native Method)
```

**Comment by BishopGIS** [comment]
May be several threads started:
```
00:07 D/nextgismobile: createLayerStorage: layer_20160902161522212
00:07 I/LayerFillService: Received start id 1: Intent { act=com.nextgis.maplibui.ADD_FILL_LAYER_TASK cmp=com.nextgis.mobile/com.nextgis.maplibui.service.LayerFillService (has extras) }
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, db_perimet REAL, history TEXT, county TEXT, entry TEXT, name TEXT, gid INTEGER, variety_1 TEXT, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
00:07 D/nextgismobile: create layer table: CREATE TABLE IF NOT EXISTS layer_20160902161522212 ( _id INTEGER PRIMARY KEY, _geom_2 BLOB, _geom_4 BLOB, _geom_6 BLOB, _geom_8 BLOB, _geom_10 BLOB, _geom_12 BLOB, _geom_14 BLOB, _geom_16 BLOB, _geom_18 BLOB, _geom BLOB, agroproc_1 TEXT, agroproc_2 TEXT, agroproc_3 TEXT, agrotechni TEXT, dt_date TIMESTAMP, crop_2 TEXT, crop_1 TEXT, history TEXT, db_perimet REAL, county TEXT, name TEXT, entry TEXT, variety_1 TEXT, gid INTEGER, variety_2 TEXT, pesticides TEXT, db_area REAL );
00:07 D/nextgismobile: init layer fields
```

**Comment by 4eRTuk** [comment]
Fix in 73f19d7d5f8a4e5b39f6bc9d904cef8279fe3368
